src/main/js/components/contextMenu/ColumnContextMenu.jsx

```diff
@@ -90,8 +90,6 @@ class ColumnContextMenu extends React.Component {
 }
 
 ColumnContextMenu.propTypes = {
-  x: React.PropTypes.number.isRequired,
-  y: React.PropTypes.number.isRequired,
```

Comment:
They are optional in the GenericContextMenu, to which we compose ColumnContextMenu, and by setting them implicitly by CSS in ColumnContext we don't need the at all there.